lib/Twig/Extension/Escaper.php

```diff
@@ -77,7 +84,7 @@ public function getDefaultStrategy($filename)
     {
         // disable string callables to avoid calling a function named html or js,
         // or any other upcoming escaping strategy
-        if (!is_string($this->defaultStrategy) && is_callable($this->defaultStrategy)) {
```

Comment:
I removed that check because it hides problems. If somebody passed an array that is not callable, it will just return the array as-is (which is wrong). Now it will raise an error properly.